docs/en/patterns/forms.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Forms layout
+table_of_contents: true
+---
+
+# Form inline and stacked styles
+
+By applying the classes ```.p-form--inline .p-form--stacked``` and wrapping any form control in ```.p-form__group``` you can change the layout style of any form and to fit a design purpose.
+
+<a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/forms/"
```

Comment:
Again this link should be forms/form but the doc shouldn't be in a directory on its own.